Domain/Entity/Book.cs

```diff
@@ -22,9 +25,4 @@ public Book(Guid id, string title, string author, long isbn, string genre, DateT
         PublicationYear = publicationYear;
     }
 
-    public override string ToString()
-    {
-        return $"Title: {Title}, Author: {Author}, ISBN: {Isbn}, Genre: {Genre}, Publication Year: {PublicationYear:yyyy-MM-dd}";
-    }
-
```

Comment:
Was this method being used by some logger? Maybe we can use it in the future

Reply:
All prints were made in another class so as not to depend on the model.

Reply:
great